src/Elmish.WPF/Binding.fs

```diff
@@ -1686,9 +1698,10 @@ type Binding private () =
   ///
   ///   Only use this if you are unable to use some kind of <c>SelectedValue</c> or
   ///   <c>SelectedIndex</c> property with a normal <see cref="twoWay" /> binding.
-  ///   This binding is less type-safe and will throw at runtime if
-  ///   <paramref name="subModelSeqBindingName" /> does not correspond to a
-  ///   <see cref="subModelSeq" /> binding, or if the inferred <c>'id</c> type
+  ///   This binding is less type-safe.
+  ///   It will throw when initializing the bindings if <paramref name="subModelSeqBindingName" />
+  ///   does not correspond to a <see cref="subModelSeq" /> binding,
+  ///   and it will throw at runtime if if the inferred <c>'id</c> type
```

Comment:
Would be great to have these lines normalized to ~80 characters (i.e., no line break after "type-safe.") Similar for the other overloads.

Reply:
Sure.  Auto formatted the comments in that whole file.

Reply:
Brilliant. Are you using the Rewrap VS extension?

Reply:
I used the extension for VS Code.  That is the first one I found.  I got the impression that one also exists for VS, but I didn't look into it.